Sources/MapboxDirections/MapMatching/Match.swift

```diff
@@ -3,9 +3,38 @@ import CoreLocation
 import Polyline
 import struct Turf.LineString
 
-extension CodingUserInfoKey {
-    static let tracepoints = CodingUserInfoKey(rawValue: "com.mapbox.directions.coding.tracepoints")!
+public enum Weight: Equatable {
```

Comment:
Where can I find some docs or something to learn more about what "Weight" is?

Reply:
Good catch, I wrote a documentation comment for the enum.